pkgs/applications/video/gnome-subtitles/default.nix

```diff
@@ -0,0 +1,91 @@
+{ stdenv
+, lib
+, fetchFromGitLab
+, autoreconfHook
+, pkg-config
+, intltool
+, itstool
+, mono
+, yelp-tools
+, wrapGAppsHook
+, enchant2
+, glib
+, gtk3
+, gtk-doc
+, gtk-sharp-3_0
+, gst_all_1
+, libxml2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-subtitles";
+  version = "1.6";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = pname;
+    rev = "${pname}-${version}";
+    sha256 = "1crmmcx32i6ca7dlr3xhnc7vgv9jhlpwh6hxhv2fl1x1zbasf42z";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    gtk-doc
+    autoreconfHook
+    mono
+    intltool
+    itstool
+    pkg-config
+    yelp-tools
+  ];
+
+  buildInputs = [
+    enchant2
+    gtk3
+    libxml2
+    glib
+    gtk-sharp-3_0
+  ] ++ (with gst_all_1; [
+    gstreamer
+    gst-plugins-base
+    (gst-plugins-good.override {
+      gtkSupport = true;
+    })
+    gst-plugins-bad
+  ]);
+
+  preConfigure = ''
+    intltoolize
+  '';
+
+  # The unreleased master branch looks after libenchant2 by default.
+  # So this would not be needed in the next release.
+  prePatch = ''
+    substituteInPlace src/GnomeSubtitles/Execution/gnome-subtitles.exe.config \
```

Comment:
Does this patch apply? https://gitlab.gnome.org/GNOME/gnome-subtitles/-/commit/bae6f6d853c70ede461440ce8002913ea1472089

Reply:
That patch does not apply.... I believe this would apply: https://gitlab.gnome.org/GNOME/gnome-subtitles/-/compare/gnome-subtitles-1.6...bae6f6d853c70ede461440ce8002913ea1472089 but can't find a way to turn it into a patch... adding `.patch` at the end does not work

Reply:
There is no longer need for a patch. Marking as solved.